extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcConfigurationMetadata.java

```diff
@@ -3,6 +3,15 @@
 import io.vertx.core.json.JsonObject;
 
 public class OidcConfigurationMetadata {
+    private static final String ISSUER = "issuer";
+    private static final String TOKEN_ENDPOINT = "token_endpoint";
+    private static final String INTROSPECTION_ENDPOINT = "introspection_endpoint";
+    private static final String TOKEN_INTROSPECTION_ENDPOINT = "token_introspection_endpoint";
```

Comment:
We should just ignore this one.

Reply:
Hi @pedroigor 
I'd be happy to do it, but `vertx-auth` supports it - which is why `integration-tests/oidc-wiremock` passes (it uses `token-introspection-endpoint`). Would you mind if we delay removing this property, may be till Quarkus 2 ? At the moment I'd like to minimize any side-effects with the migration...
That said, if this property was only used with very old Keycloak versions which Quarkus users have a low chance of using then indeed we can avoid this extra check
What do you think ?

Reply:
@pedroigor I guess we can indeed continue focusing on the standard path - if someone reports any problem in 1.13.0.CR1 then I'll put it back, let me update